source/_integrations/plugwise.markdown

```diff
@@ -224,45 +207,223 @@ action: `climate.set_preset_mode`
 
 Available options include: `home`, `vacation` (Anna only), `no_frost`, `asleep` & `away`.
 
-Example:
+Automation example:
+
+```yaml
+# Example automation changing the active (or currently set by schedule) preset
```

Comment:
Please make sure this matched our YAML style guides.

Also move the comment/introduction comment text out. It is better to introduce it in normal text instead.